cloudify_cli/config/parser_config.py

```diff
@@ -668,43 +666,42 @@ def parser_config():
                             },
                             '--json': json_events_argument()
                         },
-                        'help': 'Display Events for different executions',
+                        'help': 'Display events for different executions',
                         'handler': cfy.events.ls
                     }
                 }
             },
             'executions': {
-                'help': "Manage Cloudify's Executions",
+                'help': "Handle workflow executions",
                 'sub_commands': {
                     'get': {
                         'arguments': {
                             '-e,--execution-id': execution_id_argument(
-                                hlp='The id of the execution to get'
+                                hlp='The ID of the execution to get'
                             )
                         },
-                        'help': 'Get an execution by its id',
+                        'help': 'Get an execution by its ID',
                         'handler': cfy.executions.get
                     },
                     'list': {
                         'arguments': {
                             '-d,--deployment-id': deployment_id_argument(
-                                hlp='The Deployment id to list executions for'
+                                hlp='The deployment ID to list executions for'
                             ),
                             '--system-workflows': {
                                 'dest': 'include_system_workflows',
                                 'action': 'store_true',
-                                'help': 'Include executions of '
-                                        'system workflows'
+                                'help': 'Include executions of system workflows'
                             },
                         },
-                        'help': 'List all running executions, or all '
+                        'help': 'List all running executions on the Manager or all '
```

Comment:
Does `cfy execution list` list all **running** executions, or all executions, including those who already terminated?
If only running executions, then maybe change `or all executions for a specific deployment` to `or all **running** executions for a specific deployment`
If not only running executions, then delete the `running`.